### PR TITLE
db seeding

### PR DESCRIPTION
**Pull request by ac251**
@jawad04 Hey Jawad, when you have a chance would you mind checking out my db seeding code? Thanks very much!

### CODE REVIEW

**Comment by ac251** [comment]
Fixed a little bug, most all same. Thanks!

**Comment by ac251** [comment]
@IsaacLeeWebDev Can you take a glance at this? My db seed script is dbSeed.sh

**Comment by ac251** [comment]
Thanks @IsaacLeeWebDev !

File: fakeDataGenerator.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+const moment = require('moment');
+
+faker.seed(105);
+
+const fakeRestaurants = [];
+
+for (let i = 1; i <= 100; i++) {
+  const fakeRestaurant = {id: i};
+  const fakeWords = [];
+  const length = faker.random.number() % 3;
```
Comment (IzzyLeeDev) [inline fakeDataGenerator.js]: I think faker.random.number(3) will choose a random number between 1 and 3.

File: fakeDataGenerator.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+
+const fakeReviewTextGenerator = () => {
+  const reviewText = [];
+  const paragraphs = faker.random.number(4);
+  for (let p = 0; p <= paragraphs; p++) {
+    let paragraph = '';
+    const sentences = faker.random.number(3) + 2;
+    for (let s = 0; s <= sentences; s++) {
+      paragraph += faker.lorem.sentence();
```
Comment (IzzyLeeDev) [inline fakeDataGenerator.js]: Consider faker.lorem.paragraph(), although they should be functionally the same, unless you're shooting for exactly 2-5 sentences. I'm not sure faker.lorem.paragraph() can be passed some number of sentences, or will stay within the bounds of what your database will accept.

File: seedscript.sql [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  overall TINYINT(1) NOT NULL,
+  food TINYINT(1) NOT NULL,
+  service TINYINT(1) NOT NULL,
+  ambience TINYINT(1) NOT NULL,
+  value TINYINT(1) NOT NULL,
+  wyr BIT(1) NOT NULL,
+  text VARCHAR(4000) NOT NULL,
+  date TIMESTAMP NOT NULL,
+  partySize TINYINT NOT NULL,
+  PRIMARY KEY (id),
```
Comment (IzzyLeeDev) [inline seedscript.sql:39]: the keywords PRIMARY KEY can be written on the end of line 27 to accomplish the same effect, but this is fine if you like it better this way.

**Review by IsaacLeeWebDev — APPROVED** [review]

File: fakeDataGenerator.js (hunk 1 above)
Comment (ac251) [reply fakeDataGenerator.js]: Yeah, I mostly refactored after realizing that faker.random.number can take params. Missed that one, will do. Thanks!

File: fakeDataGenerator.js (hunk 2 above)
Comment (ac251) [reply fakeDataGenerator.js]: I do want some 2 -sentence paras, but I think you're right and might be able to do the same with faker.lorem.paragraph. Will investigate.

**Pull request merged**